lib/circlex/api/payments/bank_accounts.ex

```diff
@@ -54,6 +56,47 @@ defmodule Circlex.Api.Payments.BankAccounts do
     end
   end
 
+  @doc """
+  Create a bank account (wires) for an international bank account that supports list_bank_accounts
+
+    iex> host = Circlex.Test.start_server()
```

Comment:
You're missing `## Examples` here that would run this.

Reply:
thanks